README.adoc

```diff
@@ -60,6 +62,30 @@ You can access the following two microservices to test their availability:
 In addition, you can access a third microservice, which retrieves and aggregates all of the configuration properties and sources that have been added throughout this guide. This is available at:
 
 * http://localhost:9080/config[http://localhost:9080/config^]
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Run the following curl command to test the availability of the **system** microservice and retrieve the system information:
+```
+curl http://localhost:9080/system/properties
+```
+{: codeblock}
+
+Run the following curl command to test the availability of the **inventory** microservice and 
+retrieve the information for a list of all previously registered hosts:
+```
+curl http://localhost:9080/inventory/systems
+```
+{: codeblock}
+
+In addition, you can run the following curl command to access a third microservice, 
+which retrieves and aggregates all of the configuration properties and sources that have been added throughout this guide.
```

Comment:
...all of the configuration properties and sources that have been added throughout this guide
--->
...all of the configuration properties and sources that are added throughout this guide

Reply:
updated both static and cloud-hosted instructions